cucumber-messages/ruby/Gemfile

```diff
@@ -1,6 +1,3 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 gemspec
-
-# Pick a known-good version when running builds for cucumber-messages on JRuby.
-gem 'google-protobuf', '~> 3.2.0.2', platform: :jruby
```

Comment:
I don't know why the build doesn't pick this up, or even if the build builds on any jruby, but this change makes the gem installation phase of the build fail locally for me with jRuby 9.2.5.0 and jRuby 9.2.6.0.